solidity/interfaces/IConnextVestingWallet.sol

```diff
@@ -0,0 +1,157 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.20;
+
+import {IERC20} from '@openzeppelin/contracts/token/ERC20/IERC20.sol';
+
+interface IConnextVestingWallet {
+  /*///////////////////////////////////////////////////////////////
+                             EVENTS
+  //////////////////////////////////////////////////////////////*/
+
+  /**
+   * @notice NEXT tokens released
+   * @param _token  The address of the NEXT token
+   * @param _amount The amount of tokens released
+   */
+  event ERC20Released(address indexed _token, uint256 _amount);
+
+  /*///////////////////////////////////////////////////////////////
+                             ERRRORS
+  //////////////////////////////////////////////////////////////*/
+
+  /**
+   * @notice Permission denied
+   */
+  error NotAllowed();
+
+  /**
+   * @notice Zero address not allowed
+   */
+  error ZeroAddress();
+
+  /*///////////////////////////////////////////////////////////////
+                            CONSTANTS
+  //////////////////////////////////////////////////////////////*/
+
+  /**
+   * @notice NEXT token address
+   * @dev Mainnet address
+   * @return _nextToken The address of the NEXT token
+   */
+  function NEXT_TOKEN() external view returns (address _nextToken);
+
+  /**
+   * @notice Token launch date
+   * @dev Equals to Sept 5th 2023
+   * @return _timestamp The timestamp of the token launch
+   */
+  function NEXT_TOKEN_LAUNCH() external view returns (uint64 _timestamp);
+
+  /**
+   * @notice 1 month in seconds (on average)
+   * @return _timedelta The timedelta of one month
+   */
+  function ONE_MONTH() external view returns (uint64 _timedelta);
+
+  /**
+   * @notice 1 year in seconds
+   * @return _timedelta The timedelta of one year
+   */
+  function ONE_YEAR() external view returns (uint64 _timedelta);
+
+  /**
+   * @notice Sept 5th 2023 in seconds
+   * @return _timedelta The timestamp of Sept 5th 2023
+   */
+  function SEPT_05_2023() external view returns (uint64 _timedelta);
+
+  /**
+   * @notice Vesting cliff duration
+   * @dev 1 month cliff
+   * @return _timedelta The timedelta of the cliff duration
+   */
+  function VESTING_CLIFF_DURATION() external view returns (uint64 _timedelta);
+
+  /**
+   * @notice Vesting duration including one month of cliff
+   * @dev 13 months duration
+   * @return _timedelta The timedelta of the vesting duration
+   */
+  function VESTING_DURATION() external view returns (uint64 _timedelta);
+
+  /**
+   * @notice Vesting warmup time
+   * @dev 11 months offset
```

Comment:
i'm not sure if `@dev` comments are correct in the interface, because they're talking about the immutable variable that's on the contract, it doesn't make sense to assume in the interface that this will return `11 months offset`

i'd move the `@dev` comments to the contract